cartofacade/carto_facade.go

```diff
@@ -173,3 +173,126 @@ func (cf *CartoFacade) start(ctx context.Context, activeBackgroundWorkers *sync.
 		}
 	}()
 }
+
+// Initialize calls into the cartofacade C code.
+func (cf *CartoFacade) Initialize(ctx context.Context, timeout time.Duration, activeBackgroundWorkers *sync.WaitGroup) error {
+	cf.start(ctx, activeBackgroundWorkers)
+
+	requestParams := map[RequestParamType]interface{}{}
+	untyped, err := cf.request(ctx, initialize, requestParams, timeout)
```

Comment:
How about we pull out all the empty requestParame into a package level constant?
```
const emptyRequestParams := map[RequestParamType]interface{}{}
```

```suggestion
	untyped, err := cf.request(ctx, initialize, emptyRequestParams, timeout)
```